Extensions/DialogueTree/JsExtension.js

```diff
@@ -619,6 +653,40 @@ module.exports = {
       .getCodeExtraInformation()
       .setFunctionName('gdjs.dialogueTree.compareVariable');
 
+    extension
+      .addCondition(
+        'CompareDialogueStateNumberVariable',
+        _('Compare dialogue state number variable'),
+        _(
+          'Compare dialogue state number variable. Use this to trigger game events via dialogue variables.'
+        ),
+        _('Dialogue state number variable _PARAM0_ is equal to _PARAM1_'),
+        _('Dialogue Tree (experimental)'),
+        'JsPlatform/Extensions/yarn24.png',
+        'JsPlatform/Extensions/yarn32.png'
+      )
+      .addParameter('string', _('State variable'), '', false)
+      .addParameter('expression', _('Equal to'), '', false)
+      .getCodeExtraInformation()
+      .setFunctionName('gdjs.dialogueTree.compareVariable');
+
+    extension
+      .addCondition(
+        'CompareDialogueStateBooleanVariable',
+        _('Compare dialogue state boolean variable'),
+        _(
+          'Compare dialogue state variable. Use this to trigger game events via dialogue variables.'
+        ),
+        _('Dialogue state boolean variable _PARAM0_ is equal to _PARAM1_'),
+        _('Dialogue Tree (experimental)'),
+        'JsPlatform/Extensions/yarn24.png',
+        'JsPlatform/Extensions/yarn32.png'
+      )
+      .addParameter('string', _('State variable'), '', false)
+      .addParameter('yesorno', _('Equal to true'), '', false)
```

Comment:
I think `trueorfalse` is better than `yesorno`

Reply:
updated now